### ISSUE TITLE
I can't get the precision When I train MapNet in 'chess' of '7Scenes'

### ISSUE THREAD

**Issue by zhuixunforever** [NVlabs/geomapnet#34]
I train mapnet with:
--dataset 7Scenes --scene chess --config_file configs/mapnet.ini --model mapnet --device 0 --learn_beta --learn_gamma

and the 'mapnet.ini' is:
;;;
;Copyright (C) 2018 NVIDIA Corporation.  All rights reserved.
;Licensed under the CC BY-NC-SA 4.0 license (https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode).
;;;

[training]
n_epochs = 300
batch_size = 20
do_val = yes
seed = 7
shuffle = yes
num_workers = 5
snapshot = 5
val_freq = 5
max_grad_norm = 0

[optimization]
opt = adam
lr = 1e-4
weight_decay = 0.0005
;momentum = 0.9
;lr_decay = 0.1
;lr_stepvalues = [60, 80]

[logging]
visdom = no
print_freq = 20

[hyperparameters]
beta = -3.0
gamma = -3.0
dropout = 0.5
skip = 10
variable_skip = no
real = no
steps = 3
color_jitter = 0.0

but I eval the 300 model with:
--dataset 7Scenes --scene chess --model mapnet --weights logs/7Scenes_chess_mapnet_mapnet_learn_beta_learn_gamma/epoch_300.pth.tar --config_file configs/mapnet.ini --val
and get:
Error in translation: median 0.0992 m,  mean 0.1284 m
Error in rotation: median 3.6509 degrees, mean 4.5607 degree

but in your pretrained 'epoch_250.pth.tar', like your paper, that is:
Error in translation: median 0.0833 m,  mean 0.1273 m
Error in rotation: median 3.2206 degrees, mean 4.3601 degree.

can you give me some advices?

**Comment by samarth-robo** [comment]
Hi @zhuixunforever , small differences of training outcomes can be hard to debug. Sorry for the late reply. I would try playing with the `batch_size`, `beta`, and `gamma` parameters, and check the performance for checkpoints saved at other epochs.

**Comment by samarth-robo** [comment]
Please re-open if you still have questions.
